vortexasdk/client.py

```diff
@@ -36,7 +36,7 @@ class VortexaClient:
     _N_THREADS = 6
     _MAX_ALLOWED_TOTAL = int(1e6)
 
-    def __init__(self, **kwargs):
+    def __init__(self: "VortexaClient", **kwargs: Dict) -> None:
```

Comment:
nit: should we have `Any` rather than `Dict` here? or are we expecting individual kwargs to be only dict ?https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values